### PR TITLE
Uma 60

### PR DESCRIPTION
**Pull request by manojkrishna1991**

### CODE REVIEW

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.less [hunk 1]
```diff
@@ -133,3 +136,20 @@
     }
 }
 
+.empty-nodebox {
+    display: flex;
+    margin: 10px 12px;
+    width: 180px;
+    border: 1px solid #D7D7D7;
```
Comment (BenjaminEckardt) [inline src/multi-level-checkbox-editor/multi-level-checkbox-editor.less]: I think we have variables for these colors

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.less [hunk 2]
```diff
@@ -133,3 +136,20 @@
...
+    background: #FFFFFF;
```
Comment (BenjaminEckardt) [inline src/multi-level-checkbox-editor/multi-level-checkbox-editor.less]: Same here, pls use vars

**Comment by BenjaminEckardt** [comment]
@manojkrishna1991  pls add @MostafaBalata

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx [hunk 3]
```diff
@@ -96,6 +103,11 @@ class MultiLevelCheckboxEditor extends React.Component<Props, State> {
                                     this.getInfoText()}
                             </React.Fragment>
                         ))}
+                        {this.props.maxBoxCount && this.state.selectedNodeIds.length + 1 < this.props.maxBoxCount ? (
```
[the remark this answers is not in the document]
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx]: sounds good will do it

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx [hunk 4]
```diff
@@ -80,7 +81,13 @@ class MultiLevelCheckboxEditor extends React.Component<Props, State> {
                         {nodes.map((node, level) => (
                             <React.Fragment key={node.id}>
                                 {hasChildren(node) && (
-                                    <div style={{ width: `${100 / this.props.levelHeaderLabels.length}%` }}>
+                                    <div
+                                        style={{
+                                            width: this.props.maxBoxCount
+                                                ? '300px'
```
[the remark this answers is not in the document]
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx]: removed it and added a clas style

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx]: done

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.less [hunk 5]
```diff
@@ -133,3 +136,20 @@
...
+    padding: 20px;
+    max-height: 250px;
+    box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
+
+    > p {
```
[the remark this answers is not in the document]
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.less]: done

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.less (hunk 1 above)
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.less]: changed

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.less (hunk 2 above)
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.less]: changed

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx [hunk 6]
```diff
@@ -104,6 +115,10 @@ class MultiLevelCheckboxEditor extends React.Component<Props, State> {
         );
     }
 
+    private isEmptyBox() {
+        return this.props.maxBoxCount && this.state.selectedNodeIds.length + 1 < this.props.maxBoxCount;
```
Comment (MostafaBalata) [inline src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx]: I don't really understand the meaning of Empty Box, why would you call something empty if it returns a value?
imagine `maxBoxCount = 6`
`selectedNodeIds.length = 2` 
then `2 + 1 < 6 ` returns `true` which represented as empty.

Maybe I miss understand something, can you explain to me?

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.less [hunk 7]
```diff
@@ -1,155 +1,160 @@
...
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.05);
+
+  .empty-nodebox-content {
+    margin: auto;
+    text-align: left;
+  }
+}
 
+.arrange-type-box {
+  width: 300px ! important;
```
Comment (BenjaminEckardt) [inline src/multi-level-checkbox-editor/multi-level-checkbox-editor.less]: Please avoid important rules. They spread very easily and are hard to control.

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx (hunk 6 above)
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx]: yeah true the name should be changed thanks :)

File: src/multi-level-checkbox-editor/multi-level-checkbox-editor.less (hunk 7 above)
Comment (manojkrishna1991) [reply src/multi-level-checkbox-editor/multi-level-checkbox-editor.less]: done removed it

**Pull request merged**